docs/src/components/FigmaLink.tsx

```diff
@@ -8,7 +8,8 @@ export const FigmaLink = ({ href }: { href: URL }) => (
       borderColor: 'background.light',
       borderRadius: '10px',
       textAlign: 'center',
-      justifySelf: 'end',
+      width: 'max-content',
+      ml: 'auto',
```

Comment:
i think we should position the figma link button on the left side on mobile devices and change in line 15 `mt: -2rem`. to `mt: -1rem`